splunk_add_on_ucc_framework/install_python_libraries.py

```diff
@@ -30,26 +32,37 @@ class SplunktaucclibNotFound(Exception):
     pass
 
 
+class WrongSplunktaucclibVersion(Exception):
+    pass
+
+
 class CouldNotInstallRequirements(Exception):
     pass
 
 
-def _subprocess_call(
+class InvalidArguments(Exception):
+    pass
+
+
+def _subprocess_run(
```

Comment:
If we separate this refactoring code out of this PR, then the feature code to add check on the `splunktaucclib` version would be super small and straightforward 😄

Reply:
Ok, I will split it into two separate PR's but I will create branch with splunktaucclib from this one to have proper logic.